db-migration/src/main/resources/db/migration/V2__add_accounts_table.sql

```diff
@@ -1,26 +0,0 @@
-CREATE TABLE roles
```

Comment:
This commit: 7f4ef3cbda860b39f2e939344db4d03daaed7952 is unecessery beacause it's already reverted from develop branch. Please remove this commit.

Reply:
tip: git rebase -i HEAD~2

Reply:
Done